recipes/r-mapi/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set version = '1.0.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-mapi
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/mapi_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/mapi/mapi_{{ version }}.tar.gz
+  sha256: 8530add41950907f7099f0548168962d2ff94cb6cae5fb0dfd5a3a57ee1e9556
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcpp >=0.10.0
+    - r-data.table >=1.10
+    - r-pbapply >=1.3
+    - r-sf >=0.5
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rcpp >=0.10.0
+    - r-data.table >=1.10
+    - r-pbapply >=1.3
+    - r-sf >=0.5
+
+test:
+  commands:
+    - $R -e "library('mapi')"           # [not win]
+    - "\"%R%\" -e \"library('mapi')\""  # [win]
+
+about:
+  home: https://www1.montpellier.inra.fr/CBGP/software/MAPI/
+  license: GPL-2.0-or-later
```

The cran meta data says `License: GPL (>= 3)`, so:
```suggestion
  license: GPL-3.0-or-later
```